molecule/default/unmount.yml

```diff
@@ -0,0 +1,33 @@
+---
+- name: Unmount /etc/resolv.conf
+  hosts: all
+  become: true
+  become_method: ansible.builtin.sudo
+  tasks:
+    - name: >-
```

Comment:
Would you please separate each of these tasks such as we do in `tasks/main.yml`? I just find the lack of separation unpleasant when reading through a list of tasks.

Reply:
Please see commit 2e803337eac0997b764446e4d715b4b6d1483d0a.

Reply:
Appreciated!